apps/server/src/modules/authorization/authorization.service.spec.ts

```diff
@@ -243,8 +208,18 @@ describe('authorization.service', () => {
 			expect(loader.loadEntity).lastCalledWith(AllowedAuthorizationEntityType.User, userId);
 		});
 
+		it('Should call ReferenceLoader.getUserWithPermissions.', async () => {
+			const userId = new ObjectId().toHexString();
+			const user = userFactory.buildWithId({}, userId);
+			loader.loadEntity.mockResolvedValue(user);
+			const res = await service.getUserWithPermissions(userId);
```

Comment:
In our docs for testing it says: "Your test should be structured in three seperate areas, each distinguished by at least an empty line" (https://hpi-schul-cloud.github.io/schulcloud-server/additional-documentation/nestjs-application/testing.html) 

So you should add two empty lines here.